docs/explanations/architecture/modularity.md

```diff
@@ -78,6 +78,8 @@ These are packages used in development mode to help developers with daily tasks
 
 ## Editor Packages
 
+![Post Editor Modules Architecture](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/explanations/architecture/assets/modules.png)
```

Comment:
I noticed that there's a number of assets that we link to like that (using a GitHub link). I assume we explored alternatives in the past. Anyone know why we settled on this? This doesn't sound ideal to me because we always link to the "trunk" version of the files.

Reply:
We looked at uploading files to dot-org and linking them (I think a few assets _may_ be linked that way?) but ultimately didn't go for it because it would have required coordinating auth permissions for people.

It might be possible to use relative links, which I _think_ would make GitHub use the current branch/commit view instead of trunk, but I don't think that would translate to DevHub without some extra work on the meta side.

Reply:
Should we create another repository in the WordPress organization to keep all images and use the same permissions for users as they exist in Gutenberg?

There are 35 images referenced in the documentation at the moment that are hosted together with the source code. The image included in this PR alone is nearly 1 MB, so those assets (and their previous versions kept in git history) might have some bigger impact on the download size when cloning the repository.